webdriver-spec.html

```diff
@@ -5036,9 +5041,23 @@
    <dd><p>Toggle <var>element</var>’s <a>selectedness</a> state.
 
    <dt><a><code>option</code> element</a>
-   <dd><p>Set <var>element</var>’s <a>selectedness</a> state to true.
+   <dd>
+     <ol>
+       <li><p>Let <var>parent node</var> be the calling
```

Comment:
You probably want to focus on the `<a>container</a>`. `parentNode` might be an `optgroup` no?